scripts/mail_resp.py

```diff
@@ -17,37 +17,21 @@ def main(args):
 
     operator="denis.moreno@scilifelab.se"
     summary={}
-    email={u'Anna Konrad':'anna.konrad@scilifelab.se',
-            u'Bahram Amini':'bahram.amini@scilifelab.se',
-            u'Britta Lotstedt':'britta.lotstedt@scilifelab.se',
-            u'Carolina Bonilla':'carolina.bonilla@scilifelab.se',
-            u'Chuan Wang':'chuan.wang@scilifelab.se',
-            u'Christian Natanaelsson':'christian.natanaelsson@scilifelab.se',
-            u'Francesco Vezzi':'francesco.vezzi@scilifelab.se',
-            u'Helena Samuelsson':'helena.samuelsson@scilifelab.se',
-            u'Helena Zajac':'helena.zajac@scilifelab.se',
-            u'Joel Gruselius':'joel.gruselius@scilifelab.se',
-            u'Jun Wang':'jun.wang@scilifelab.se',
-            u'Kicki Holmberg':'kicki.holmberg@scilifelab.se',
-            u'Lina Sylwan':'lina.sylwan@scilifelab.se',
-            u'Mario Giovacchini':'mario.giovacchini@scilifelab.se',
-            u'Marianna Sjogren':'marianna.sjogren@scilifelab.se',
-            u'Nemanja Rilakovic':'nemanja.rilakovic@scilifelab.se',
-            u'Mattias Oskarsson':'mattias.oskarsson@scilifelab.se',
-            u'Par Lundin':'par.lundin@scilifelab.se',
-            u'Phil Ewels':'phil.ewels@scilifelab.se',
-            u'Remi-Andre Olsen':'remi-andre.olsen@scilifelab.se',
-            u'Senthilkumar Paneerselvam':'senthilkumar.panneerselvam@scilifelab.se',
-            u'Senthilkumar Panneerselvam':'senthilkumar.panneerselvam@scilifelab.se',
-            u'Simone Picelli':'simone.picelli@scilifelab.se',
-            u'Sverker Lundin':'sverker.lundin@scilifelab.se'
-            }
     project_types=['Bcl Conversion & Demultiplexing (Illumina SBS) 4.0','Illumina Sequencing (Illumina SBS) 4.0', 
     'MiSeq Run (MiSeq) 4.0','Cluster Generation (Illumina SBS) 4.0','Denature, Dilute and Load Sample (MiSeq) 4.0', 'Aggregate QC (DNA) 4.0','Aggregate QC (RNA) 4.0', 'Project Summary 1.3']
 
     def clean_names(name):
         return name.replace(u"\u00f6", "o").replace(u"\u00e9", "e").replace(u"\u00e4", "a")
 
+    def get_email(fullname):
+        names=fullname.split(" ")
```

Comment:
ok.. still you can try to clean names everytime... it affects only people with strange names.... what about people with two names (i.e., name secondname lastname) ? maybe we do not have this case....